src/integration/java/uk/nhs/prm/repo/ehrtransferservice/repo_incoming/EhrRequestTest.java

```diff
@@ -41,23 +46,33 @@ class EhrRequestTest {
     private TransferTrackerDbUtility transferTrackerDbUtility;
 
     @Autowired
-    private QueueUtility queueUtility;
+    private SQSQueueUtility SQSQueueUtility;
 
     @Value("${aws.repoIncomingQueueName}")
     private String repoIncomingQueueName;
 
+    @Value("${activemq.inboundQueue}")
+    private String inboundQueue;
+
     @Value("${gp2gpMessengerAuthKey}")
     private String gp2gpMessengerAuthKey;
 
     private static final String NHS_NUMBER = "9798548754";
     private static final UUID INBOUND_CONVERSATION_ID = UUID.fromString("ce3aad10-9b7c-4a9b-ab87-a9d6521d61b2");
+    private static final UUID SMALL_EHR_INBOUND_CONVERSATION_ID = UUID.fromString("ff27abc3-9730-40f7-ba82-382152e6b90a");
     private static final String NEMS_MESSAGE_ID = "eefe01f7-33aa-45ed-8aac-4e0cf68670fd";
     private static final String NEMS_EVENT_LAST_UPDATED = "2017-11-01T15:00:33+00:00";
     private static final String SOURCE_GP = "B14758";
 
+    @BeforeEach
+    void beforeEach(@Value("${inboundTimeoutSeconds}") String inboundTimeoutSeconds) {
```

Comment:
Surely this value will need to be defined within `application.properties` in the `test` directory?